### ISSUE TITLE
About resuming the model

### ISSUE THREAD

**Issue by wxxxxxxh** [thstkdgus35/EDSR-PyTorch#175]
Hi, thank you for sharing your code!
When I resume the training process at a certain epoch or the latest epoch, I find that the model will begin from the epoch 1 rather than our set.
Can you provide some solutions about these error?
Thanks very much.

**Comment by thstkdgus35** [comment]
Hello.

Would you provide the training script used for resuming your training?

I know there are some minor bugs in resuming, but I think we can fix this issue by using an appropriate script.

Thank you!

**Comment by wxxxxxxh** [comment]
Hi
Thanks for your reply.
Below is the script which I used to resume my training:
`nohup python -u main.py --scale 4 --save temp_model_vgg --load temp_model_vgg --resume 220 --model WDSR_B --loss 0.9*L1+0.1*VGG22 --ext img --batch_size 16 --test_every 1000 --dir_data ../../youku/images/ --data_train YOUKU --data_test YOUKU --n_resblocks 32 --n_feats 256 --n_GPUs 4 --save_models > youku_L1VGG.log 2>&1 &`
Thank you!
